Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+# Use the Rails Config gem so we can use Settings.yml and its friends
+gem 'config'
+
+# Use Postgres as our database
+gem 'pg'
+
+# Use Jbuilder to generate JSON
+gem 'jbuilder', '~> 2.5'
+
+# useful for debugging, even in prod
+gem 'pry-byebug' # Adds step-by-step debugging and stack navigation capabilities to pry using byebug
+gem 'pry-rails'  # use pry as the rails console shell instead of IRB
+
+# Use Puma as the app server
+gem 'puma', '~> 3.7'
+gem 'rails', '~> 5.1'
+
+
+group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri]
+  # Call 'binding.pry' anywhere in the code to stop execution and get a pry console
+  gem 'rspec-rails', '~> 3.6'
+  gem 'rails-controller-testing'
```

Comment:
add comment:  rails-controller-testing # allows use of assigns in tests

or delete gem until we need it?